pkgs/tools/video/blackmagic-desktop-video/default.nix

```diff
@@ -86,7 +101,7 @@ stdenv.mkDerivation rec {
     cp -r $unpacked/usr/share/doc/desktopvideo $out/share/doc
     cp $unpacked/usr/lib/*.so $out/lib
     cp $unpacked/usr/lib/systemd/system/DesktopVideoHelper.service $out/lib/systemd/system
-    cp $unpacked/usr/lib/blackmagic/DesktopVideo/DesktopVideoHelper $out/bin/
+    cp -r $unpacked/usr/lib/blackmagic/DesktopVideo/* $out/bin/
```

Comment:
this also puts a bunch of libraries into bin, which I'd like to avoid. It's probably necessary to copy them individually.

Reply:
Binaries are linked to them as ./<lib>.so. This was the easiest way to get it working. Haven't tried moving them to lib, maybe elf patcher would pick them up